Solved Issue #9334. Added unpin feature for rewards list in tasks.

klezt opened the pull request:
Fixes #9334 

### Changes
Added Unpin button the same way it is used featured items in market. Message pops up if reward cannot be unpinned.

UUID: efcfbb29-939d-437f-bebe-058bdef04997

klezt committed:
Solved Issue #9334. Added unpin feature for rewards list in tasks.

paglias commented on website/client/components/tasks/column.vue:
```diff
@@ -58,11 +58,50 @@
 <style lang="scss" scoped>
   @import '~client/assets/scss/colors.scss';
 
+  .badge-svg {
```
this css isn't already available somewhere else?

paglias replied:
cc @negue

klezt replied:
@paglias I tried to import them using "@import '/client/components/shops/market/index/badge-svg.css';", however the class is not recognized.

paglias replied:
`/client/components/shops/market/index/badge-svg.css';` doesn't exist as a file, if you meant the `.vue` file then the css is only available inside that .vue file. You should move it inside a .scss file in client/assets/scss and then you can use it everywhere

paglias commented on website/client/components/tasks/column.vue:
```diff
@@ -586,6 +627,15 @@ export default {
+    togglePinned (item) {
+      try {
+        if (!this.$store.dispatch('user:togglePinnedItem', {type: item.pinType, path: item.path})) {
+          this.$parent.$parent.text(this.$t('unpinnedItem', {item: item.text}));
```
you can import the notifications mixin into this component directly

klezt committed:
Does not reuse same css classes, imports them from pin.scss. For the togglePin function the notification.

klezt committed:
Added pin.scss to index.scss.

klezt commented:
@paglias Sorry for the issues, hope it is alright now.

paglias commented on website/client/components/shops/market/index.vue:
```diff
@@ -234,6 +234,7 @@
 <style lang="scss">
   @import '~client/assets/scss/colors.scss';
   @import '~client/assets/scss/variables.scss';
+  @import '~client/assets/scss/pin.scss';
```
If you could remove this too it would be great, thanks :)

klezt committed:
Deleted unnessecary import.

paglias commented:
@klezt hey, I've merged the PR but looks like your UUID is not the correct one, can you check it? So that I can award your contributor tier, thanks!

klezt commented:
@paglias UUID: 016566f9-6c31-4a07-b4e3-541a7d9bb077

paglias commented:
tier awarded :)
